src/game_ui.c

```diff
@@ -3815,7 +3815,7 @@ void hud_element_render(Gfx **dList, MatrixS **mtx, Vertex **vtxList, HudElement
     TextureHeader **textureHeader3;
     TextureHeader *textureHeader2;
     TextureHeader *textureHeader;
-    ObjectSegment *objSegment;
+    Camera *objSegment;
```

Comment:
suggestion: rename to camera or activeCamera given it's usage below:
objSegment = cam_get_active_camera();